ocs_ci/ocs/elasticsearch.py

```diff
@@ -245,3 +306,38 @@ def _copy(self, es):
             for doc in result["hits"]["hits"]:
                 log.debug(f"Going to write : {doc}")
                 es.index(index=ind, doc_type="_doc", body=doc["_source"])
+
+    def dump(self, target_path):
```

Comment:
Please use a better function name for better readability.

Reply:
renamed.